README.md

```diff
@@ -54,25 +54,34 @@ Register the plugin on your application and add the [access token](https://www.s
 
 > You need to declare the plugin use and its options in `gatsby-config.js`
 
+`gatsby-config.ts`
```

Comment:
Is there a reason to have `gatsby-config.ts` twice? Or maybe it wasn't on purpose? 🤔

Reply:
Yes @Dawntraoz The one you pointed out shows Gatsby plugin configuration for our Gatsby SDK💡
This part below focuses on for Partial Hydration config:

https://github.com/storyblok/gatsby-source-storyblok/pull/340/commits/9c5f70730b6fc4579e41adcb6aff86bbab11d4aa#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R311-R318